test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/Multiple.Schema.OpenApi.json

```diff
@@ -2623,7 +2623,7 @@
           "content": {
             "application/json": {
               "schema": {
-                "type": "String"
+                "type": "object"
```

Comment:
should there also be an additionalproperties section specifying the type of the values.

Reply:
We should make this a dictionary by adding additionalproperties - https://swagger.io/docs/specification/data-models/dictionaries/

Reply:
@peombwa  Would this 'type' be valid for PowerShell? 

![image](https://user-images.githubusercontent.com/40403681/89923505-607f6700-dc09-11ea-94a6-5b2418183ede.png)

Reply:
That's a valid type in PowerShell, and we will treat it as a dictionary. Another alternative would be:
``` yml
type: object
additionalProperties:
  type: object
```

Reply:
I would prefer it free-form so that other users may be able to specify their own types.

Reply:
Resolved